changelog.d/1107.breaking.md

```diff
@@ -0,0 +1,2 @@
+Fixes the `to_float` function to return an error instead of "infinity" when parsing a string outside
+the f64::MAX / f64::MIN bounds.
```

Comment:
```suggestion
Fixes the `to_float` function to return an error instead of `f64::INFINITY` when parsing [non-normal](https://doc.rust-lang.org/std/primitive.f64.html#method.is_normal) numbers.
```